src/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+}
+
+.card {
+  font-family: Roboto;
```

Comment:
Which font family will be displaying if google fonts not responding?
Please, add some alternative common families, like `Arial` or `sens-serif`.